preflight-checks/nccl-allreduce/nccl_allreduce/__main__.py

```
@@ -94,6 +95,18 @@ def run() -> int:
 
+    store_only = cfg.processing_strategy == pb.ProcessingStrategy.STORE_ONLY
+    exit_code = _run_benchmark_flow(cfg)
+
+    if exit_code != 0 and store_only:
+        log.warning("Check failed (STORE_ONLY — not blocking pod)")
+        return NCCLError.SUCCESS.value.exit_code
+    return exit_code
+
+
+def _run_benchmark_flow(cfg: Config) -> int:
+    """Execute the benchmark and return an exit code."""
```

Ruff (0.15.10)

[warning] No blank lines allowed after function docstring (found 1)

Remove blank line(s) after function docstring

(D202)